src/snippets/muiAvatar.ts

```diff
@@ -0,0 +1,8 @@
+import snip from '../snip'
+
+export const description = 'Material-UI <Avatar>'
+export const body = snip`
+<Avatar>
```

Comment:
I think I'll plan to add an `muiAvatarImage` snippet for cases like `<Avatar alt="Remy Sharp" src="/static/images/avatar/1.jpg" />`